client/e2e/testData/dataConstants/ShowingInfoConstants.ts

```diff
@@ -18,17 +18,17 @@ export const SHOWING_INFO_2: ShowingInfo = {
 
 export const SHOWING_INFO_3: ShowingInfo = {
   showingDate: '2021-09-16',
-  showingTime: '',
+  showingTime: '12:31',
   showingQuantity: '101',
-  showingWholeDate: '',
-  showingDateTime: '',
+  showingWholeDate: 'Thu, Sep 16 2021Time: 12:31',
```

Comment:
whats the difference between showingWholeDate and shoing DateTime?

Reply:
Also why is showing date time different format across SHOWING_INFOs? (look at line 16).